src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/on_system_request_notification.cc

```diff
@@ -1,5 +1,5 @@
 /*
- Copyright (c) 2018, Ford Motor Company
+ Copyright (c) 2016, Ford Motor Company
```

Comment:
why this one goes back to 2016, but others stay at 2018

Reply:
@yang1070 
fixed c244b2f